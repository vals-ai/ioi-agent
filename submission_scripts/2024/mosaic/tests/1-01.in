version https://git-lfs.github.com/spec/v1
oid sha256:2d1dd1e79c3349df84d3ac64cd261ff4d4d66690f6ad510d3bdbffd28e0d1359
size 122
