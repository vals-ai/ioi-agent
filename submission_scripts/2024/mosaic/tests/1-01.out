version https://git-lfs.github.com/spec/v1
oid sha256:604f14661afcc8e1a07db434041f0b524ea8a4365d6d576d8bebdca36c225765
size 56
